src/builders/rollup/utils.ts

```diff
@@ -1,3 +1,11 @@
+import { dirname, resolve } from "pathe";
+import {
+  sys,
+  findConfigFile,
+  readConfigFile,
+  parseJsonConfigFileContent,
+} from "typescript";
```

Comment:
We should avoid loading `typescript` package as it adds runtime overhead.

Reply:
I have considered this issue and come up with two alternative solutions:

1. Implement a lightweight `tsconfig.json` parsing logic that only parses the `paths` and `baseUrl` fields to infer the alias.

2. Use a lightweight alternative library (e.g., https://github.com/privatenumber/get-tsconfig) (or implement one ourselves) to parse the complete `tsconfig.json` configuration, which can not only be used for alias inference but also for other configurations, such as automatically passing it to esbuild's `tsconfigRaw` (which can resolve https://github.com/unjs/unbuild/issues/309), thereby eliminating the need to duplicate existing content from `tsconfig.json` in configuration files.

I'd love to hear your feedback! 🤔